### PR TITLE
Sort items based on index and position

### PR DESCRIPTION
**Pull request by henriettemoe**
Get correct tab order for accessibility

### CODE REVIEW

File: src/components/Grid/Grid.tsx [hunk 1]
```diff
@@ -38,6 +38,25 @@ export const Grid: React.FC<GridProps> = ({
   const [itemShowingDialog, setItemShowingDialog] =
     useState<CommonItemType | null>(null);
 
+  const sortItems = (a: TopicMapItemType, b: TopicMapItemType): number => {
+    // Sort after index first
+    if (a.index && b.index) {
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:43]: The index might be `0` which is falsy, however should be sorted before 1, in my opinion.

```suggestion
    if (a.index != null && b.index != null) {
```

File: src/components/Grid/Grid.tsx [hunk 2]
```diff
@@ -38,6 +38,25 @@ export const Grid: React.FC<GridProps> = ({
...
+      return a.index < b.index ? -1 : 1;
+    }
+    if (a.index && !b.index) {
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:46]:
```suggestion
    if (a.index != null && b.index == null) {
```

File: src/components/Grid/Grid.tsx [hunk 3]
```diff
@@ -38,6 +38,25 @@ export const Grid: React.FC<GridProps> = ({
...
+      return -1;
+    }
+    if (!a.index && b.index) {
```
Comment (boyum) [inline src/components/Grid/Grid.tsx:49]:
```suggestion
    if (a.index == null && b.index != null) {
```

**Review by boyum — APPROVED** [review]
Nice!

File: src/components/Grid/Grid.tsx (hunk 1 above)
Comment (henriettemoe) [reply src/components/Grid/Grid.tsx]: Yes, thanks!

**Pull request merged**